starboard/build/toolchain/linux/clang_lld_toolchain.gni

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build/toolchain/gcc_toolchain.gni")
+
+template("clang_lld_toolchain") {
```

Comment:
Feel free to to in a later cl . Not fully sure why this template is needed instead of reusing gcc_toolchain. Also the name is a little confusing like why clang_lld ? Can you add a comment clarifying that ?

Reply:
It uses gcc_toolchain, it's just a specific variant of that toolchain that is a clang toolchain with an lld linker